pages/_app.tsx

```diff
@@ -41,43 +39,12 @@ function MyApp({ Component, pageProps }: AppProps) {
         <meta name="msapplication-TileColor" content="#ffffff" />
         <meta name="theme-color" content="#000000" />
       </Head>
-      <DefaultSeo
-        openGraph={{
-          type: 'website',
-          locale: 'en_US',
-          url: `https://${process.env.NEXT_PUBLIC_VERCEL_URL}/landing-page-screenshot.png`,
-          site_name: 'Developer DAO Academy',
-          title: 'Developer DAO Academy',
-          description:
-            'Academy is an open-source education platform created by the Developer DAO.',
-          images: [
-            {
-              url: `https://${process.env.NEXT_PUBLIC_VERCEL_URL}/landing-page-screenshot.png`,
-              alt: 'Developer DAO Academy',
-              type: 'image/png',
-            },
-          ],
-        }}
-        twitter={{
-          handle: '@devdao_academy',
-          site: '@devdao_academy',
-          cardType: 'summary_large_image',
-        }}
-        additionalLinkTags={[
-          {
-            rel: 'icon',
-            href: '/favicon/favicon.ico',
-          },
-        ]}
-      />
-      <Box p="1.25em" px="5%" mx={{ base: '2rem', md: '6rem', lg: '10rem' }}>
-        <Header />
-        <MDXProvider components={Components}>
+      <MDXProvider components={Components}>
+        <Layout>
           <Component {...pageProps} />
-        </MDXProvider>
-        <Footer />
+        </Layout>
         <Analytics />
```

Comment:
@okhaimie-dev are you sure if the <MDXProvider> has to wrap the <Analytics> component from vercel?

Reply:
https://vercel.com/docs/concepts/analytics/quickstart

From what I understand, yes, it is okay for the `<MDXProver /> to wrap the `<Analytics />` component. I am confident that it work that way rn.

Reply:
Hey Okhai!

To let You know the one that resolves the conversations is the one who commented 😉
That way this mate is validating that the comment was addressed